site/specs-docs-only/messagingInternational.json

```diff
@@ -0,0 +1,202 @@
+                                }
+                            }
+                        }
+                    },
+                    "default": {
+                        "description": "error",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/Error"
+                                }
+                            }
+                        }
+                    }
+                },
+                "x-codegen-request-body-name": "post data"
+            }
+        }
+    },
+    "components": {
+        "schemas": {
+            "SMSRequest": {
+                "required": [
+                    "auth",
+                    "dcs",
+                    "dlrMask",
+                    "dlrUrl",
+                    "receiver",
+                    "sender",
+                    "text",
+                    "type"
+                ],
+                "type": "object",
+                "properties": {
+                    "type": {
+                        "type": "string",
+                        "description": "Type of SMS (text, wsi)"
+                    },
+                    "sender": {
+                        "type": "string",
+                        "description": "SMS Sender"
+                    },
+                    "receiver": {
+                        "type": "string",
+                        "description": "SMS Receiver (mobile phone number in international format)\n"
+                    },
+                    "dlrMask": {
+                        "type": "integer",
+                        "description": "Delivery report mask:\n  1 - Delivered,\n  2 - Undelivered,\n  4 - Buffered\n  8 - Sent to SMSC\n  16 - Rejected.\nRecommended value is 19.\n"
+                    },
+                    "dlrUrl": {
+                        "type": "string",
+                        "description": "URL where to send callbacks\n"
+                    },
+                    "flash": {
+                        "type": "boolean",
+                        "description": "Use true for flash message, false otherwise"
+                    },
+                    "text": {
+                        "type": "string",
+                        "description": "if type=text this is message text"
+                    },
+                    "dcs": {
+                        "type": "string",
+                        "description": "if type=text this is encoding, GSM or UCS"
+                    },
+                    "url": {
+                        "type": "string",
+                        "description": "if type is wsi, URL for WAP Push"
+                    },
+                    "title": {
+                        "type": "string",
+                        "description": "if type is wsi, title of message"
+                    },
```

Comment:
I'm not sure where we received our info from for the previous docs but it mentions the only type supported by Bandwidth is "text". Assuming true, we wouldn't require these.